core/src/test/java/io/paradoxical/cassieq/unittests/TestBase.java

```diff
@@ -121,6 +121,7 @@ protected TestQueueContext setupTestContext(String queueName, int bucketSize) {
         final QueueDefinition queueDefinition = QueueDefinition.builder()
                                                                .accountName(testAccountName)
                                                                .queueName(queue)
+                                                               .allowRandomBucketReading(false)
```

Comment:
Our tests need very speciifc ordering for our asserts to work. I'll make an item to support "hand wavy" asserts that any of a particular range are consumed when we do a "readAndAck"